sensor.cpp

```diff
@@ -1029,7 +1052,7 @@ Sensor::Sensor() :
 
     previousDirection = 0xFF;
     previousCt = 0xFFFF;
-    previousSequenceNumber = 0xFF;
+    previousCommandId = 0xFF;
```

Comment:
`previousSequenceNumber = 0xFF;` 

Should likely stay :)

Reply:
Hu ? right, Don't know why I have removed removed it.
Will correct it in same time than conflict.